### ISSUE TITLE
Feature Request: Show phone number / number label

### ISSUE THREAD

**Issue by stoneubi** [SimpleMobileTools/Simple-Dialer#301]
Hi there,

it would be great if the phone number and / or the number label would be displyed at the call history overview. 

![Screenshot_2022-02-07-14-53-18-40_442be8c5b497ab9263c01542b26de11f](https://user-images.githubusercontent.com/32926093/152802207-55212e2d-c89a-4ab7-ac92-326540aa0677.jpg)

Contacts can have multiple phone numbers and actuslly you cannot see from which number the contact called.

Many thanks
Stoneubi

**Comment by tibbi** [comment]
it has just been done, will be added in the next app version 5.12.0 released soon

**Comment by nhfo** [comment]
Has it been done? I have version 5.18.0. On call history tab It still only shows contact names. 

Also "setting > calls > Group subsequent calls with the same number at the call log" does not have any effect either. I was thinking maybe when call entries are grouped it does not display number label / number.

**Comment by tibbi** [comment]
it is shown only at contacts with multiple numbers

**Comment by nhfo** [comment]
It is not the case. I have same number grouping on. There is a contact with two numbers and both numbers have different label and label is different from contact name. I called one number twice. There is one entry for it in call history tab. Then I called the other number. Now there are two entries. Both entries show contact name. A also tried clearing app storage, guessing it could be an issue related to upgrades and won't appear on fresh install, but it still does not show number label or number in call history.
